### PR TITLE
MOBDOC-23 Improve antora rendering of doc , see shiftgear.org

### PR DESCRIPTION
**Pull request by pwright**
* converted some markdown files
* moved files to antora structure
* created navigation (still draft)

### CODE REVIEW

**Comment by pwright** [comment]
@pb82 can you review this?

File: docs/modules/aerogear-push-chrome/pages/index.adoc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+The following step-by-step guides, gives you an introduction on how to
```
Comment (pb82) [inline docs/modules/aerogear-push-chrome/pages/index.adoc:1]: Is this still relevant? At the moment UPS only supports Firebase (for Android push), APNS (Apple) and Windows Push. This seems to be about pushing to Chrome using GCM.

File: docs/modules/aerogear-push-js/pages/index.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
+The https://github.com/aerogear/aerogear-unifiedpush-server.git[AeroGear
```
Comment (pb82) [inline docs/modules/aerogear-push-js/pages/index.adoc:1]: This (simple push) is no longer supported in UPS.

File: docs/modules/client-registration/pages/index.adoc [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+client library should be offered.
+
+*Goal:* A client library to register a mobile application
+(`Installation`) with the UnifiedPush Server.
+
+[[background-push-network-token]]
+Background: Push Network Token
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To receive native push notifications from a Push Network (e.g. APNs, GCM
```
Comment (pb82) [inline docs/modules/client-registration/pages/index.adoc:20]: This mentions outdated push providers (GCM, Simple Push)

**Comment by pb82** [comment]
@finp are those the docs from the old Aerogear website? Because a lot of it is outdated (we no longer support GCM and Simple Push).

**Comment by pwright** [comment]
@pb82  yes, you're right. How do you think we should proceed? should we meet up and review?

**Comment by pb82** [comment]
@finp Tbh, i'm not sure how much of the docs will be left if we remove everything that's outdated. I'm happy to meet up and help you flag the outdated docs. At some point we will probably have to rewrite or create some of them (e.g. change the Android instructions from GCM to Firebase).

**Pull request closed without merging**